### PR TITLE
[FIX] website: prevent click on readonly stat button

### LINKED ISSUE #78500: [15] website: traceback on clicking on visits button on website
Impacted versions: 15.0

https://10557349-15-0-all.runbot50.odoo.com/web#cids=1&menu_id=480&action=694&model=website.visitor&view_type=form&id=1

Steps to reproduce:
Screenshot(24)

Current behavior:

Screenshot(23)
Expected behavior: No Traceback (and open view)

Video/Screenshot link (optional):

Support ticket number submitted via odoo.com/help (optional):

### PR DESCRIPTION
**Pull request by rdeodoo**
When clicking on readonly stat button (website related), it would
sometimes raise a traceback since there is no action to redirect to.

Step to reproduce:
- Go to Website > Visitors > Visitors
- Select a Visitor to enter its form view
- Now, either:
  A. Enter edit mode
  B. Enter edit mode and discard (or save)
- Click on stat button

This will raise a traceback, since the `disabled` property is now gone.
Indeed, the framework is adding/removing that attribute when entering
or leaving edit mode.
See disableButtons()/enableButtons()` called by
_setEditMode()/_onDiscard()/..`

Note:
- With the new framework/owl, there is no TB as the `disabled`
attribute is not removed anymore.
- In 14.0, there is no issue despite the `disabled` attributed being
removed.

Fixes https://github.com/odoo/odoo/issues/78500
Closes https://github.com/odoo/odoo/pull/80884

Courtesy of @odooaktiv

### CODE REVIEW

**Comment by rdeodoo** [comment]
See https://github.com/odoo/odoo/pull/80884#pullrequestreview-909016438 for detailed explanation

File: addons/website/static/src/scss/website.backend.scss [hunk 1]
```diff
@@ -218,11 +218,15 @@
...
 .oe_stat_button {
-    &.o_stat_button_info:hover {
-        color: #666666 !important;
-        background-color: transparent !important;
-        opacity: 0.8 !important;
-        cursor: default !important;
+    &.o_stat_button_info {
+        pointer-events: none;
+
+        &:hover {
```
Comment (qsm-odoo) [inline addons/website/static/src/scss/website.backend.scss:224]: @rdeodoo I must be missing something but this should not make sense: you disable pointer events on something and then add `:hover` rules on it -> those must be dead code

**Review by qsm-odoo — CHANGES_REQUESTED** [review]

File: addons/website/static/src/scss/website.backend.scss (hunk 1 above)
Comment (rdeodoo) [reply addons/website/static/src/scss/website.backend.scss:224]: Indeed, I added that one liner without paying much attention to the already existing lines.
Plus I did not actually know `pointer-events: none` was actually preventing hover effects.

**Review by qsm-odoo — APPROVED** [review]
Given the fact that disabled stat buttons does not seem used anywhere else that these website buttons I guess it's not worth a deeper fix and we can ignore that someone may rely on those strange :hover rules we remove. It's not like it would break a flow or something :+1:

**Pull request closed without merging**